wiki/Contests/Official_support/en.md

```diff
@@ -39,12 +63,13 @@ Contests which abide by the following criteria are eligible for official support
 - **The contest must have a clearly defined judging criteria based on mapping merit.** These ensure a contest follows an understandable quality standard in line with other mapping contests.
 - **The contest's results must be determined by valuing scores from all judges equally.** Using standard deviation to determine final scores is required.[^exception-note]
   - The final score must be calculated as follows: `judge X's final score = (judge X's raw score - judge X's average raw score) / judge X's standard deviation`.
+- **The contest's host must moderate their judges' comments.** Comments which are either offensive, belittling, low effort, or otherwise inappropriate may be removed at the committee's discretion.
```

Comment:
grammatical nit

```suggestion
- **The contest's host must moderate their judges' comments.** Comments which are either offensive, belittling, of low effort, or otherwise inappropriate may be removed at the committee's discretion.
```